### ISSUE TITLE
Add event for connect and disconnect 

### ISSUE THREAD

**Issue by Bre77** [teslamotors/fleet-telemetry#188]
It would be a huge value to have the Fleet Telemetry server emit an event when a vehicle connects and disconnects, so that a third party service knows if a vehicle is awake or asleep/offline, and avoid polling the Fleet API when a vehicle is asleep or to find out when it wakes up.

I can see these events written to standard out, but they are not emitted through the event buses.

**Comment by rileymd88** [comment]
This is already supported. If the vehicle is awake and you are subscribed
to some fields then it will send data. When the car stops sending data then
it is asleep

**Comment by Bre77** [comment]
> This is already supported. If the vehicle is awake and you are subscribed
> to some fields then it will send data. When the car stops sending data then
> it is asleep

That is incorrect, certain telemetry fields are only sent under certain conditions, so when you might think a vehicle is "asleep" might actually be the vehicle in park.

**Comment by patrickdemers6** [comment]
Closing as this has been implemented.

Related though, there are reliability issues: #244
